src/System.Text.Json/src/System/Text/Json/Serialization/JsonSerializer.Write.Stream.cs

```diff
@@ -11,11 +11,28 @@ namespace System.Text.Json.Serialization
 {
     public static partial class JsonSerializer
     {
+        /// <summary>
+        /// Convert the value and write it to the <see cref="System.IO.Stream"/>.
```

Comment:
Convert the value... to UTF-8 encoded JSON text
Or ... write ... as UTF-8 encoded JSON text

Reply:
Changed to `Convert the provided value to UTF-8 encoded JSON text and write it to the...`